test/react-native/features/support/maze.all.cfg

```diff
@@ -0,0 +1,3 @@
+--no-tunnel
+--aws-public-ip
```

Comment:
These two should only be in place on Buildkite (BrowserStack will be used locally with the secure tunnel).

Reply:
True, but all the React-native tests use Buildkite AFAICT

Reply:
In principle engineers could be running tests on their local machine when writing new tests, even if it's awkward to build the test fixture.